recipes/conda-build/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.21.5" %}
+
+package:
+  name: conda-build
+  version: {{ version }}
+
+source:
+  fn: conda-build-{{ version }}.tar.gz
+  url: https://github.com/conda/conda-build/archive/{{ version }}.tar.gz
+  sha256: af90f4d086377bfa9e9d2be590c1d14c79df58937fb12102b67bf0a5e03032c8
+
+build:
+  number: 0
+  script: python setup.py install
+  entry_points:
+    - conda-env = conda_env.cli.main:main
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - conda >=4.1
+    - jinja2
```

Comment:
Thanks. Missed that. Though it seems like this is [required]( https://github.com/conda/conda-build/blob/1.21.5/conda.recipe/meta.yaml#L19 ) for all build or do you just mean the removal will only by on Windows or all Unixes. Sorry a bit confused.